src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -99,214 +113,343 @@ const GET_PROFILE_HEADING_QUERY = {
-							<SchoolIcon className="iconScaling" />
-						</Grid>
-						<Grid className="rowPadding" item xs={5}>
-							<Typography className="rowText">{userInfo.school?.name}</Typography>
-						</Grid>
-						<EmailRow />
-					</>
-				);
-			case "INSTRUCTOR":
-				return (
-					<>
-						<IDRow width={12} />
-						<PhoneRow width={12} />
-						<EmailRow />
-					</>
-				);
-			case "PARENT":
-				return (
-					<>
-						<IDRow />
-						<Grid className="rowPadding" item xs={1}>
-							<MoneyIcon className="iconScaling" />
-						</Grid>
-						<Grid className="rowPadding" item xs={5}>
-							<Typography className="rowText">${userInfo.balance}</Typography>
-						</Grid>
-						<PhoneRow width={12} />
-						<EmailRow />
-					</>
-				);
-			default:
-				return (
-					<>
-						<IDRow width={12}/>
-						<PhoneRow width={12}/>
-						<EmailRow/>
-					</>
-				);
-		}
-	};
+  const handleCloseReset = () => {
+    setResetOpen(false);
+  };
+
+  const isAdmin = loggedInAuth.accountType === USER_TYPES.admin;
+  const isUser = loggedInAuth.user.id === userInfo.user.id;
+
+  const renderEditandAwayButton = () => (
+    <Grid container item xs={4}>
+      {(isAdmin || isUser) && (
+        <>
+          <Grid component={Hidden} item mdDown xs={12}>
+            <EditIcon className="editIcon" />
+            <div className="editResetDiv">
+              <ResponsiveButton
+                component={Link}
+                to={`/form/${userInfo.accountType.role}/${userInfo.user.id}`}
+                className="edit"
+              >
+                Edit Profile
+              </ResponsiveButton>
+              {isAdmin &&
+                (userInfo.accountType.role === "student" ? (
+                  <ResponsiveButton
+                    className="resetStudent"
+                    disabled
+                    onClick={handleClickOpen}
+                  >
+                    Reset Password
+                  </ResponsiveButton>
+                ) : (
+                  <ResponsiveButton className="reset" onClick={handleClickOpen}>
+                    Reset Password
+                  </ResponsiveButton>
+                ))}
+            </div>
+            <ResetDialog />
+          </Grid>
+          <Grid component={Hidden} item lgUp xs={12}>
+            <Button
+              component={Link}
+              to={`/form/${userInfo.accountType.role}/${userInfo.user.id}`}
+              variant="outlined"
+            >
+              <EditIcon />
+            </Button>
+          </Grid>
+        </>
+      )}
+    </Grid>
+  );
+
+  const ResetDialog = () => (
+    <>
+      <Dialog
+        open={open}
+        onClose={handleClosePassword}
+        maxWidth="xs"
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title" className="center dialog-padding">
+          {"Do you want to reset this user's password?"}
```

Comment:
remove curly brackets and add "disableTypography" for the proper heading to be in effect

Reply:
Done thank you!